bool all_motors_in_position
bool lookup_table_move_in_progress